apps/courtofhumanity/.env.example

```
@@ -0,0 +1,18 @@
+# Database and authentication
+DATABASE_URL="postgresql://user:password@host-pooler/database?sslmode=require"
+NEXTAUTH_SECRET="generate-a-random-32-byte-secret"
+NEXTAUTH_URL="http://localhost:3010"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)